public/static/docs/user-guide/troubleshooting.md

```diff
@@ -3,6 +3,8 @@
 In this section we provide help for some of the problems that DVC user might
 stumble upon.
 
+<!-- This file uses a special engine feature for the following headers, so that a custom anchor link is used. Just add {#custom-anchor} after each title: -->
```

Comment:
no multi-line support for the comment?

Reply:
Ah yeah sorry. I just had to add the first new line after `<!--` and then it rewraps it 😬. Fixed